client/state/selectors/get-current-query-arguments.js

```diff
@@ -15,6 +15,6 @@ import { get } from 'lodash';
  * @param {Object} state - global redux state
  * @return {Object} current state value
  */
-export const getCurrentQueryArguments = state => get( state, 'ui.route.query.current', null );
+export const getCurrentQueryArguments = state => get( state, 'ui.route.query.current', {} );
```

Comment:
The default `{}` object will be a new instance on each selector call, and that can cause unneeded React rerenders. It would be better to return a constant object that's defined just once. But it's a very theoretical nit, as this case will probably never happen in practice. Just pointing out a common selector bug that sometimes has real consequences 🙂

Reply:
Done, thanks for the tips!